docs/rbac.rst

```diff
@@ -0,0 +1,53 @@
+.. _rbac:
+
+RBAC Configuration
+******************
+
+To enhance security, the `edit` `ClusterRoleBinding` has been removed from
+the Kanister Helm Chart. Users now need to create their own `Role` / `RoleBinding`
+in the application's namespace to grant the necessary permissions to
+Kanister's Service Account.
+
+Creating a RoleBinding with edit ClusterRole
+============================================
+
+To allow Kanister to perform backup/restore operations in a specific
+namespace, create a `RoleBinding` that assigns the `edit` `ClusterRole`
+to Kanister's Service Account:
+
```

Comment:
I think we should mentioned that `edit` is a system created cluster role to make sure users don't confused about where did `edit` cluster role come from.

Reply:
Updated